dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -1,15 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 import "..\AppNotifications\AppNotifications.idl";
 
 namespace Microsoft.Windows.AppNotifications.Builder
 {
-    [contractversion(1)]
+    [contractversion(2)]
     apicontract AppNotificationBuilderContract {}
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationHeader
+    {
+        AppNotificationHeader();
+        AppNotificationHeader(String id, String title);
```

Comment:
Are `Id` and `Title` required?

Wondering why they warrant a non-default ctor but other runtimeclasses don't